src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,103 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
too complex logic, let's overwrite it. Create 3 methods:
1. Read from a file, return read data
2. Form statistic, here you should calculate the salary for each employee and form a report using StringBuilder, return String here.
3. Write to file, pass formed statistic here